semgrep_output_v1.atd

```diff
@@ -820,6 +854,47 @@ type ci_scan_results <ocaml attr="deriving show"> = {
 
 
 
+type parsing_stats <ocaml attr="deriving show"> = {
+  targets_parsed: int;
+  num_targets: int;
+  bytes_parsed: int;
+  num_bytes: int;
+}
+
+type ci_scan_complete_stats <ocaml attr="deriving show"> = {
+  findings: int;
+  errors: cli_error list;
+  total_time: float;
+  unsupported_exts: (string * int) list
+    <json repr="object">
+    <python repr="dict">
+    <ts repr="map">;
+  lockfile_scan_info: (string * int) list
+    <json repr="object">
+    <python repr="dict">
+    <ts repr="map">;
+  parse_rate: (string * parsing_stats) list
+    <json repr="object">
+    <python repr="dict">
+    <ts repr="map">;
+  (* This is EngineType from python, which is different from engine_kind used in this file *)
+  ?engine_requested: string option;
+}
+
+type ci_scan_dependencies <ocaml attr="deriving show"> =  (string * found_dependency list) list
+    <json repr="object">
+    <python repr="dict">
+    <ts repr="map">
+
+type ci_scan_complete <ocaml attr="deriving show"> = {
```

Comment:
should we try to be more specific about req vs resp, eg `ci_scan_complete_response` or something?

Reply:
Yeah that seems reasonable